dotlottie-rs/src/dotlottie_player.rs

```diff
@@ -136,6 +138,8 @@ impl DotLottieRuntime {
             markers: MarkersMap::new(),
             active_animation_id: String::new(),
             active_theme_id: String::new(),
+            width: 0,
+            height: 0,
```

Comment:
Can we add these properties to the Renderer instead of DotLottieRuntime? since they are more renderer related